pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2690,8 +2690,7 @@ type StorageScopeSpec struct {
 	// This is needed for OSD flapping where OSD daemons are marked down more than 5 times in 600 seconds by Ceph.
 	// Preventing the OSD pods to restart immediately in such scenarios will prevent Rook from marking OSD as `up` and thus
 	// peering of the PGs mapped to the OSD.
-	// The interval defaults to 24 hours if no value is provided. User needs to manually restart the OSD pod if they manage to fix
-	// the underlying OSD flapping issue before the restart interval.
+	// User needs to manually restart the OSD pod if they manage to fix the underlying OSD flapping issue before the restart interval.
```

Comment:
Let's mention the new default is to disable the sleep if it's 0. Also, need to update the `flappingRestartIntervalHours` in the doc ceph-cluster-crd.md.